bin/build-metadata.js

```diff
@@ -214,6 +214,18 @@ function removeAllNumberKeys( obj ) {
 	return _.omitBy( obj, function( val, key ) { return /^\d+$/.test( key ); } );
 }
 
+function removeRegionCodeAndCountryDialCodeIfSameWithCountryDialCode( countryData ) {
+	for ( var key in countryData ) {
```

Comment:
Yes, we can probably move to that at some point, but the rest of the code in the file uses the old school approach, so in order to keep the consistency I'm not going to change this right now :)